R/bimm_predict.R

```diff
@@ -16,20 +16,23 @@ bimm_predict <- function(object,
                          new_data,
                          type = 'new_obs'){
 
-  if(type == 'new_sub'){
-    p <- predict(object = object$model_ml,
+  p <- predict(object = object$model_ml,
                  data = new_data,
                  type = 'response')
+
+  if(type == 'new_sub'){
     return(p$predictions[, 2, drop=TRUE])
   }
 
   if(type == 'new_obs'){
 
-    tmp <- data.frame(._x_. = object$fun_pred_ml(object$model_ml))
+    tmp <- data.frame(._x_. = p$predictions[, 2, drop=TRUE])
```

Comment:
I think I made a bad choice naming this `tmp` back in the day, because now I don't remember what it does. We could re-name it in this PR if you'd like something more intuitive.

Reply:
This is the dataset for predicting stuff from the GLMM model. Sure, I can rename to reflect that. Thanks for the suggestion!